frontend/lib/map/map/map.dart

```diff
@@ -47,13 +48,13 @@ class _MapState extends State<Map> implements MapController {
               ? Map.userLocationZoomLevel
               : Map.initialZoomLevel),
       styleString: config.mapStyleUrl,
-      myLocationEnabled: widget.myLocationEnabled,
+      myLocationEnabled: true,
       myLocationTrackingMode: widget.myLocationEnabled
           ? MyLocationTrackingMode.Tracking
           : MyLocationTrackingMode.None,
       onMapCreated: _onMapCreated,
       onMapClick: _onMapClick,
-      compassViewPosition: CompassViewPosition.BottomRight,
+      compassEnabled: false,
```

Comment:
:+1:

Reply:
Hm, I think I would rather show the mapbox styled compass instead of no compass at all.

Reply:
Hmm okay, I will try to place in the top right corner with some margin then :)